packages/db/prisma/extensions/stats.ts

```diff
@@ -32,7 +32,9 @@ export const statsExtension = Prisma.defineExtension((prisma) =>
           });
         },
         populate(dailyDatePeriod?: DatePeriodLike) {
-          const { from, to } = toDailyDatePeriod(dailyDatePeriod);
+          const { from: from = MIN_DATE, to: to = new Date() } = dailyDatePeriod
```

Comment:
IMO it was clearer as it was before, when the function did the checks and set the defaults

Reply:
I encountered some errors while running tests, and it took me a while to trace them back to this function setting default date values.

To prevent similar unexpected side effects in the future, I’d prefer to align this function with the`toDailyDate` logic, focusing solely on date normalization without setting default values.